mysql-test/suite/rpl/t/bug84415.test

```diff
@@ -0,0 +1,265 @@
+-- source include/master-slave.inc
```

Comment:
@laurynas-biveinis as discussed over slack : upper_bound stays, but wait for mts checkpoint and + 1 to upper bound go away.